### PR TITLE
feat: Improve docker image

### PR DESCRIPTION
**Pull request by reneleonhardt**
* Run with unprivileged user to improve security
* Allow running in user-provided timezone instead of UTC
* Improve build context cache by moving ARG down
* Use https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys

### CODE REVIEW

**Comment by coufalja** [comment]
Amazing contribution @reneleonhardt!

**Comment by reneleonhardt** [comment]
> Amazing contribution @reneleonhardt!

Feels good to be appreciated, tell my boss to pay me fairly for all the extra work 😄
I added the change to the quickstart docs.

File: .dockerignore [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
-.github
-.vscode
-.git
-docs
+.idea/
+.github/
+.vscode/
+.git/
+docs/
 helm
```
Comment (reneleonhardt) [inline .dockerignore:6]: I couldn't find a file or folder named `helm`, should I remove this entry or is it created during build or local testing?

Comment (coufalja) [reply .dockerignore:6]: You can remove it the helm chart is now located in separate repository https://github.com/jamf/regatta-helm Also if I may ask you could you create complementary PR into the regatta-helm to use correct fsGroup in security context going forward?

**Review by coufalja — APPROVED** [review]

**Pull request merged**